addon/utils/is-object.ts

```diff
@@ -1,7 +1,5 @@
-// @flow
-
 import { typeOf } from '@ember/utils';
 
-export default function isObject(val /*: mixed */) {
+export default function isObject(val: object) {
   return typeOf(val) === 'object' || typeOf(val) === 'instance';
```

Comment:
need to look into why this is throwing a typescript error:

```
addon/utils/is-object.ts(4,10): error TS2367: This condition will always return 'false' since the types '"function" | "regexp" | "array" | "error" | "filelist" | "date" | "class" | "instance"' and '"object"' have no overlap.
```

Reply:
this error is no longer being thrown